packages/discovery-provider/src/api/v1/models/users.py

```diff
@@ -36,6 +36,14 @@
         "handle": fields.String(required=True),
         "id": fields.String(required=True),
         "is_verified": fields.Boolean(required=True),
+        "twitter_handle": fields.String,
+        "instagram_handle": fields.String,
+        "tiktok_handle": fields.String,
+        "verified_with_twitter": fields.Boolean(required=True),
```

Comment:
this is a lot of very sparse data to send back to the client. can we just do an enum?

Reply:
I think sparse is better here. I started with enum but it does not handle all situations. It ends up removing current features where we check whether verified by twitter, instagram, tiktok. I know we chatted about this, but I ended doing the simple thing where we keep all features 1:1. we can chat sync if youd like

Reply:
what about verified enum for the user table? 😬

Reply:
Synced offline, there are users that are verified with multiple right now. Will keep as is and consider for future.